.tests/check-diagrams

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+diagram_dir="./docs/attestation/"
+diagrams=$(make --quiet -C "$diagram_dir" diagrams)
+
+status=0
+for diagram in $diagrams ; do
+	# Truncate the file extension (.png, .svg, etc)
+	source=${diagram%????}
+
+	source_date=$(git log -1 --date=unix --format="%cd" -- "$diagram_dir$source")
+	diagram_date=$(git log -1 --date=unix --format="%cd" -- "$diagram_dir$diagram")
+
+	if [[ $diagram_date -lt $source_date ]]; then
+		echo "$diagram_dir$diagram"
```

Comment:
Near as I can tell, this just emits the file path of any offending diagrams. Could you add a brief explanation of the problem and what needs to be done to fix it? That would be more helpful when a developer encounters a test failure.

[Example](https://github.com/enarx/enarx/blob/fbaa92856f4b6b275ee72e492902221f92f5ae02/.tests/misc-eof#L8) of what I mean.